.github/files/renovate-post-upgrade.sh

```diff
@@ -41,14 +42,19 @@ fi
 #pnpm config set --global store-dir /tmp/renovate/cache/others/pnpm
 #composer config --global cache-dir /tmp/renovate/cache/others/composer
 
-# Do the pnpm and changelogger installs.
+# Do the pnpm install. Turn off some strictness settings to make it more likely this will work.
 cd "$BASE"
+echo 'strict-peer-dependencies = false' >> .npmrc
+echo 'strict-dep-builds = false' >> .npmrc
 pnpm install
-cd projects/packages/changelogger
+git restore .npmrc
```

Comment:
I guess it doesn't hurt and it probably ensures a "clean" state, but what happens without the `git restore` on line 24? 🤔

Reply:
Renovate does its own mangling of `.npmrc`, which the restore on line 24 cleans up.

I think that, with our particular renovate config, all renovate does is add an extra newline at the end of `.npmrc`. Other things it might do are applying [the npmrc config option](https://docs.renovatebot.com/configuration-options/#npmrc), removing any line starting with "package-lock", and removing any line containing "=${".